issue_credential/README.md

```diff
@@ -219,6 +219,12 @@ Message Format:
     "goal_code": "<goal-code>",
     "comment": "some comment"
   },
+  "formats": [
```

Comment:
Is this a valid DIDComm message?
As far as I know, `format` is not a message header https://identity.foundation/didcomm-messaging/spec/#message-headers

Reply:
I agree this `formats` mapping is extraneous as the format is found on the actual attachment

Reply:
if needed, it should be included inside the body I think.